docs/api/view.md

```diff
@@ -62,9 +62,13 @@ it becomes the topmost view.
 
 If the view passed as a parameter is not a child of this view, this method is a no-op.
 
-#### `view.setBounds(bounds)`
+#### `view.setBounds(bounds[, options])`
 
 * `bounds` [Rectangle](structures/rectangle.md) - New bounds of the View.
+* `options` Object (optional) - Options for setting the bounds.
+  * `animate` boolean | Object (optional) - If true, the bounds change will be animated. If an object is passed, it can contain the following properties:
+    * `duration` Integer (optional) - Duration of the animation in milliseconds.
+    * `easing` string (optional) - Easing function for the animation. Can be `linear`, `ease-in`, `ease-out`, or `ease-in-out`. Default is `linear`.
```

Comment:
It's probably better if you could list possible values as a child list

* easing string (optional) - ...
  * 'linear'
  * 'ease-in'
  * ...

Also, could you write default value on both properties.